src/perl/Permabit/Statistics/Definitions.pm

```diff
@@ -0,0 +1,1029 @@
+##
+# A hash which describes the statistics from auto-generated code.
+#
+# @synopsis
+#
+#     use Permabit::Statistics::Definitions
+#     $VDO_STATISTICS_DEFINITIONS->{StatisticsVersion}
+#
+# @description
+#
+# This package provides a hashref which describes all of the
+# fields from a set of statistics.
+#
+# ATTENTION!!! ATTENTION!!! ATTENTION!!! ATTENTION!!! ATTENTION!!!
+#
+# If you add new statistics, be sure to update the following files:
+#
+# ../../../c++/vdo/base/statistics.h
+# ../../../c++/vdo/base/message-stats.c
+# ../../../c++/vdo/base/pool-sysfs-stats.c
+# ../../../c++/vdo/user/messageStatsReader.c
+# ../../../c++/vdo/user/vdoStatsWriter.c
+#
+##
```

Comment:
The perl file is really downstream form the .c c changes, so I think we don't need the file list here. This would mean cutting lines 13-23.